scraping_google_images.py

```diff
@@ -0,0 +1,108 @@
+# tutorial form https://www.youtube.com/watch?v=7KhuEsq-I8o
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+import requests
+import io
+from PIL import Image
+import time
+import os
+
+# Driver for chrome = ChromeDriver
+PATH = r'C:\Users\Andrew Wilhelm\OneDrive - Florida Gulf Coast University\4th year 2022-2023\Fall 2022\CNT 4104 Software Proj Comp Networks\seleniumImage\Chrome Driver\chromedriver.exe'
```

Comment:
@Andrew-Wilhelm The process of pulling the ChromeDriver needs to be automated. I recommend reading the following documentation to assist with this task: https://www.selenium.dev/documentation/webdriver/getting_started/install_drivers/